src/cli_args.rs

```diff
@@ -234,7 +234,7 @@ fn ignore<T>(_: T) {}
 
 fn parse_line_number_style<'a>(
     config: &mut AppConfig,
-    value: Option<&'a str>,
+    value: Option<&'a String>,
```

Comment:
it feel weird that you have to change this type.
what you could do is change the call site to `maybe_line_number_style.map(|s| &s[..])`

Reply:
That looks like overcomplicating, but I'll do if you really want. I'll leave a comment in the commit that I was asked by the maintainer to do that if you're okay with that, because I personally would not like that.

Reply:
By comment I mean inside the commit message